stdlib/asm/crypto/stark/ood_frames.masm

```diff
@@ -16,100 +16,103 @@ use.std::crypto::stark::random_coin
 #!
 #! 1. n, m and l are the widths of the main segment, auxiliary segment and constraint composition
 #!    traces, respectively.
-#! 2. S_i are the evaluations of columns in the main segment, auxiliary segment and constraint composition
-#!    at the the out-of-domain point z.
-#! 3. T_i are the evaluations of columns in the main segment, auxiliary segment and constraint composition
-#!    at the the out-of-domain point gz.
+#! 2. S_i are the evaluations of columns in the main segment, auxiliary segment and constraint
+#!    composition at the the out-of-domain point z.
+#! 3. T_i are the evaluations of columns in the main segment, auxiliary segment and constraint
+#!    composition at the the out-of-domain point gz.
 #! 4. alpha is the randomness used in order to build the DEEP polynomial.
 export.load_and_horner_eval_ood_frames
```

Comment:
Let's add the expected input/output stack state to doc comments here.

Reply:
done